src/sinks/aws_cloudwatch_logs/mod.rs

```diff
@@ -64,6 +66,21 @@ pub struct CloudwatchLogsSinkConfig {
     pub assume_role: Option<String>,
 }
 
+#[cfg(test)]
+fn default_config(e: Encoding) -> CloudwatchLogsSinkConfig {
```

Comment:
minor nit: if this is for a test can we put it in a `mod tests`?

Reply:
You'll notice there are two `mod tests` in this file, so to avoid having two of these fn, I made it shared.